package.json

```diff
@@ -127,6 +127,7 @@
     "morgan": "^1.5.3",
     "multer": "^1.1.0",
     "node-media-server": "^2.1.4",
+    "node-uploadx": "^4.3.0",
```

Comment:
To avoid including aws and google packages, it is better to import only `@uploadx/core`

Reply:
Thanks for the heads up!

Reply:
Is is resolved?

Reply:
Yes, in [`db16ad7` (#3933)](https://github.com/Chocobozzz/PeerTube/pull/3933/commits/db16ad7420ca3f3d1456e13c723c40843b05875c)

Reply:
My bad, I had misunderstood. Solved in the [`0379f07` (#3933)](https://github.com/Chocobozzz/PeerTube/pull/3933/commits/0379f0762d2002b191253f689153864b45baafa7).